modules/feature-list-view/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@uber/feature-list-view",
```

Comment:
Is this meant to be published with the `@uber/` namespace?

Reply:
Hello Chris and All,

Ib said we can't publish modules under @uber name space because we will have to have the organization registered in npm. Do we have the @mlvis organizations registered at the momnet.

Reply:
Sorry, I'm not sure I understand the question.
That's right, I don't believe we publish modules under `@uber` in our external repos that's why I pointed out this in my comment.

Reply:
Chris,

I have just created a mlvis namespace, I'd like to add our team members in there.